### DISCUSSION TITLE
Is there a way to know where a templaterCreateNote was called from?

### DISCUSSION THREAD

**Discussion by AWegnerGitHub** [mProjectsCode/obsidian-meta-bind-plugin#419] [Q&A]
I have a button that looks similar to this in my Daily Note template.

```meta-bind-button
label: New Note
hidden: false
class: ""
tooltip: "Create a new note"
id: "note-btn"
style: primary
actions:
  - type: templaterCreateNote
    templateFile: Templater/Note.md
    openNote: true
```

In the created note, I'd like to have a link back to the page that created this new note. Is there a way to know where the button was clicked as pass it so that I can utilize that in the template?

The process flow I'm looking for:
1. Click "Open today's Daily Note" button on the left ribbon. This opens the new daily note with the button within the template for the day
2. Click on "New Note" button (above) to create a new file/note
3. ??? Determine the Daily Note that created this new file and add a link back to it 

I've tried adding a `tp.system.prompt` to the template to do this, but it requires manually typing the name of the Daily Note file - exactly - to ensure the links are correct. Definitely slows down the process of creating a new note, and not something I can easily hand off to a less savvy user because it doesn't "just work" when they click the button.

**Comment by mProjectsCode** [comment]
The templater API does not allow passing of custom arguments to the template.
